content/docs/user-guide/visualization/animation/animation-editor/ragdoll/ragdoll-physics-setup.md

```diff
@@ -0,0 +1,145 @@
+| **Rotation** | ![Viewport UI ragdoll collider rotation icon](/images/user-guide/actor-animation/ragdoll-viewport-ui-collider-rotation.svg) | Activates manipulators allowing the rotation of the collider relative to its joint to be modified. This mode will be available if the joint has at least one ragdoll collider. |
+| **Dimensions** | ![Viewport UI ragdoll collider dimensions icon](/images/user-guide/actor-animation/ragdoll-viewport-ui-collider-dimensions.svg) | Activates manipulators allowing the **Height** and **Radius** of a **Capsule** collider to be modified. This mode will be available if the joint has at least one ragdoll collider and its first collider has **Capsule** geometry. |
+
+{{< note >}}
+Currently, if a joint has more than one ragdoll collider, manipulators only support editing the first collider.
+{{< /note >}}
+
+If you have more than one ragdoll collider, or if you prefer to edit values directly, you can also configure collider properties in the property editor. 
+
+1. On the **Ragdoll** tab, do the following in the collider properties:
+
+   1. Set the **Offset** and **Rotation** to move the collider to the correct location. The **Offset** and **Rotation** are relative to the joint transform.
+
+   1. Adjust the collider dimensions (for example, set the **Height** and **Radius** for a **Capsule**) to resize the collider.
+
+   ![Set the Offset, Rotation, Height, and Radius properties for the collider on the Ragdoll tab in the Animation Editor](/images/user-guide/actor-animation/ragdoll-collider-options-offset-rotation-height-radius.png)
+
+You can also set up other properties in the property editor, such as the [physics material](/docs/user-guide/interactivity/physics/nvidia-physx/materials/) and [collision filtering](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-layers/).
+
+**Saving changes**
+
+Choose **File**, **Save Selected Actors**. This saves the ragdoll data to the `.assetinfo` file for the character. The Asset Processor then bakes the ragdoll data into the `.actor` file.
+
+**Avoiding self-collision**
+
+The ragdoll automatically suppresses collisions between joints that are adjacent in the skeleton. This means that adjacent colliders can overlap, but a pair of colliders that are not adjacent should not intersect. If the pair of colliders intersects, they'll collide when the ragdoll is simulated. This leads to unstable behaviour, because the colliding bodies are trying to separate, while the joints are trying to keep the ragdoll together. If two joints which are not adjacent in the hierarchy have overlapping colliders, the colliders will be rendered in red, as shown below.
```

Comment:
nit: I know this is a pain but I think we're supposed to use American English for the docs so we should change 'behaviour' to 'behavior' for example (sorry if this offends your sensibilities @greerdv 😅)

Reply:
Yeah, as much as it tarnishes my very soul, I know that's the rule - just was on autopilot this time. Will update.